src/llmq/quorums_instantsend.cpp

```diff
@@ -1521,6 +1526,11 @@ bool IsInstantSendEnabled()
     return !fReindex && !fImporting && sporkManager.IsSporkActive(SPORK_2_INSTANTSEND_ENABLED);
 }
 
+bool IsInstantSendSigningEnabled()
+{
+    return !fReindex && !fImporting && sporkManager.IsSporkActive(SPORK_24_INSTANTSEND_SIGNING_ENABLED) && IsInstantSendEnabled();
```

Comment:
I think introducing `SPORK_2_INSTANTSEND_ENABLED == 1` for this limited mode instead of introducing a new spork could actually make sense, would be smth like: `0` - full mode; `1` - no mempool signing; `> now` - off. Would also match the way we handle similar issues with sporks 21 and 23.